packages/fxa-auth-server/lib/senders/emails/templates/lowRecoveryCodes/en.ftl

```diff
@@ -1,11 +1,18 @@
 # The user has a low number of valid recovery codes remaining for use
-codes-reminder-title = Low recovery codes remaining
-codes-reminder-description = We noticed that you are running low on recovery codes. Please consider generating new codes to avoid getting locked out of your account.
-codes-generate = Generate codes
-codes-generate-plaintext = { codes-generate }:
-lowRecoveryCodes-action = Generate codes
-lowRecoveryCodes-subject =
+codes-reminder-title-zero = You’re out of backup verification codes!
+codes-reminder-title-one = You’re on your last backup verification code
+codes-reminder-title-two = Time to create more backup verification codes
+
+codes-reminder-description-part-one = Backup verification codes help you restore your info when you forget your password.
+codes-reminder-description-part-two = Create new codes now so you don’t lose your data later.
+codes-reminder-description-two-left = You only have two codes left.
+codes-reminder-description-create-codes = Create new backup verification codes to help you get back into your account if you’re locked out.
+
+lowRecoveryCodes-action-2 = Create codes
+codes-create-plaintext = { lowRecoveryCodes-action-2 }:
+lowRecoveryCodes-subject-2 =
     { $numberRemaining ->
-        [one] 1 recovery code remaining
-       *[other] { $numberRemaining } recovery codes remaining
+        [zero] No backup verification codes left
+        [one] Only 1 backup verification code left
+       *[other] Only { $numberRemaining } backup verification codes left!
```

Comment:
```suggestion
        [zero] No backup authentication codes left
        [one] Only 1 backup authentication code left
       *[other] Only { $numberRemaining } backup authentication codes left!
```

Reply:
Suggesting authentication per this thread: https://mozilla-hub.atlassian.net/browse/FXA-4423

Reply:
@bcolsson, @millsoper  I can also rebase my [PR](https://github.com/mozilla/fxa/pull/14061) of this one and we can take care of this change in a single place. It might be easier to review this way.

Reply:
@dschom - I could do either so whatever works best for you two (though a single place would be easier to check).